app/controllers/games_controller.rb

```diff
@@ -5,6 +5,13 @@ def show
     @pieces = @game.pieces
     @black_player = User.find_by(id: @game.black_uid)
     @white_player = User.find_by(id: @game.white_uid)
+    if @game.is_color_in_check?(0) && @game.is_color_in_check?(1)
```

Comment:
for readability you would want to use.

Piece::COLORS[:black]
Piece::COLORS[:white] 

to denote white and black.  Once a constant is defined you should use that constant consistently throughout the app.